spritzle/resume_data.py

```diff
@@ -78,7 +78,7 @@ def __init__(self, core):
         info_hash = str(alert.handle.info_hash())
         p = Path(self.core.state_dir, alert.torrent_name + '.resume')
         r = lt.write_resume_data(alert.params)
-        r.update(self.core.torrent_data[info_hash])
+        r.update(self.core.torrent_data.get(info_hash, {}))
```

Comment:
I don't think we should change this because torrent_data for this torrent should exist when saving resume data or there is a bug somewhere.

Reply:
Yeah, this is because I used the libtorrent add and not the POST handler turns out.